website/components/DocLayout.js

```diff
@@ -393,12 +416,15 @@ export default function DocLayout({ children, toc, version, headings }) {
           </Box>
           <Link
             href={`https://github.com/robinweser/fela/edit/master/website/pages${router.pathname}.mdx`}>
-            Edit
+            Edit <VisuallyHidden>on GitHub</VisuallyHidden>
           </Link>
         </Box>
       </Box>
 
       <Box
+        as="nav"
+        id="main-navigation"
+        aria-labelledby="nav-title"
```

Comment:
This gives an accessible name to the landmark so it can be differentiated from the other ones.